WordPress/src/main/java/org/wordpress/android/ui/JetpackConnectionWebViewActivity.java

```diff
@@ -17,16 +18,49 @@
  */
 public class JetpackConnectionWebViewActivity extends WPWebViewActivity {
 
+    public enum Source {
+        STATS("stats"), NOTIFICATIONS("notifications");
+        private final String value;
+
+        Source(String value) {
+            this.value = value;
+        }
+
+        @Nullable
+        public static Source fromString(String value) {
+            if (value.equals(STATS.value)) {
```

Comment:
Looks like we need to null check `value` here.

Reply:
I'll flip the equals 👍